src/firefoxos/FileProxy.js

```diff
@@ -67,10 +69,12 @@ QUIRKS:
 
     exports.requestFileSystem = function(successCallback, errorCallback, args) {
         var type = args[0];
-        var size = args[1];
+        //var size = args[1];
```

Comment:
This var is not used, although it would be useful to leave it as a comment in terms of the function signature definition.